nss-day-cohort-29/welcome-to-nashville-valkyries
# Ab styles

# Description

Began styling site. Added background image to styles folder. Flexbox and minor styling to input fields and buttons. Added header elements to html.


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

--git fetch --all
--git checkout ab-styles
go to lib file and run grunt

check to make sure style looks clean and functionality is still intact


# Checklist:
